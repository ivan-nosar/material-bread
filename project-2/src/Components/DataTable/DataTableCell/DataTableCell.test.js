import React from 'react';
import { DataTableCell } from '../../..';

import renderer from 'react-test-renderer';

test('DataTableCell Renders', () => {
  const tree = renderer.create(<DataTableCell />).toJSON();
  expect(tree).toMatchSnapshot();
});
