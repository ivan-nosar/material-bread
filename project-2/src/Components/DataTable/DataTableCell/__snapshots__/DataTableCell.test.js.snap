// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableCell Renders 1`] = `
<View
  accessible={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderColor": "rgb(224, 224, 224)",
        "flexDirection": "row",
        "paddingHorizontal": 16,
        "paddingVertical": 4,
      },
      Object {
        "borderLeftWidth": 0,
        "borderRightWidth": 0,
        "flex": 1,
        "flexWrap": "wrap",
        "justifyContent": "flex-start",
        "minHeight": 25,
        "minWidth": 100,
      },
      Object {},
      undefined,
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "flex": 1,
          "flexShrink": 1,
          "fontSize": 13,
        },
        Object {
          "color": "rgba(0,0,0,.87)",
        },
      ]
    }
  />
</View>
`;
